src/EnergyPlus/ChillerAbsorption.cc

```diff
@@ -105,40 +105,22 @@ namespace ChillerAbsorption {
     // The Absorber program from the BLAST family of software can be used
     // to generate the coefficients for the model.
 
-    int const FlowModeNotSet(200);
-    int const ConstantFlow(201);
-    int const NotModulated(202);
-    int const LeavingSetPointModulated(203);
+    int constexpr waterIndex(1);
+    constexpr auto calcChillerAbsorption("CALC Chiller:Absorption ");
+    constexpr auto moduleObjectType("Chiller:Absorption");
 
-    int const waterIndex(1);
+    const char * fluidNameWater = "WATER";
```

Comment:
Better than string literals all over the place? 🤷‍♂️ 

The following was suggested,

```constexpr char * fluidName = "Water";``` 

but it throws warnings on my machine.

```warning: ISO C++11 does not allow conversion from string literal to 'char *const' [-Wwritable-strings]```